### PR TITLE
fix wrong color format change

### PR DESCRIPTION
**Pull request by ysku**
## What is this PR for?

- to fix bug that there is wrong color convert logic

## This PR includes

- get rid of wrong color format convert

## What type of PR is it?

Bugfix

## What is the issue?

N/A

## How should this be tested?

**TODO**

### CODE REVIEW

**Review by Plemeur — APPROVED** [review]

File: dataset.py [hunk 1]
```diff
@@ -160,9 +160,8 @@ def pull_item(self, index):
         img, boxes, labels = self.transform(
             img, self.phase, anno_list[:, :4], anno_list[:, 4])
 
-        # change channel color order from BGR to RGB
-        # and convert order from (h, w, c) to (c, h, w)
-        img = torch.from_numpy(img[:, :, (2, 1, 0)]).permute(2, 0, 1)
+        # convert order from (h, w, c) to (c, h, w)
+        img = torch.from_numpy(img).permute(2, 0, 1)
```
Comment (keigohtr) [inline dataset.py:164]: Do you mean that all incoming image format is RGB?

**Review by keigohtr — COMMENTED** [review]
One question.

File: dataset.py (hunk 1 above)
Comment (ysku) [reply dataset.py:164]: sorry for lack of the information, [here](https://github.com/abeja-inc/platform-template-object-detection/pull/12/files#diff-b24696c07a0a52292a89d29937161fcfR126) the image is converted to RGB format.

Comment (keigohtr) [reply dataset.py:164]: LGTM

**Review by keigohtr — APPROVED** [review]

**Pull request merged**